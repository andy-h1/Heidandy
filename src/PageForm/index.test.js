import React from 'react';
import { render } from '@testing-library/react';
import { PageForm } from '.';

describe('PageForm component', () => {
  it('should render', () => {
    const { container } = render(<PageForm />);
    expect(container.firstChild).toMatchSnapshot();
  });
  it.todo('should call api with correct values');
});
