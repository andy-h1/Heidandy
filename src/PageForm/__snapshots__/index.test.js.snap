// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageForm component should render 1`] = `
<form>
  <label
    for="name"
  >
    Name
    <input
      class="sc-gsTCUz bmsehX"
      id="name"
      name="name"
      placeholder="name of activity"
      type="text"
      value=""
    />
  </label>
  <label
    for="location"
  >
    Location
    <input
      class="sc-gsTCUz bmsehX"
      id="location"
      name="location"
      placeholder="location"
      type="text"
      value=""
    />
  </label>
  <label
    for="description"
  >
    Description
    <input
      class="sc-gsTCUz bmsehX"
      id="description"
      name="description"
      placeholder="description"
      type="text"
      value=""
    />
  </label>
  <label
    for="cost"
  >
    Cost
    <input
      class="sc-gsTCUz bmsehX"
      id="cost"
      name="cost"
      placeholder="cost"
      type="text"
      value=""
    />
  </label>
  <button
    class="sc-bdfBwQ bNUBxi"
    type="button"
  >
    Submit
  </button>
</form>
`;
